src/test/java/com/artipie/RepoConfigTest.java

```diff
@@ -59,13 +60,32 @@ public void readsCustom()
     @Test
     public void failsToReadCustom()
         throws URISyntaxException, ExecutionException, InterruptedException {
-        final RepoConfig config = this.readFromResource("repo-min-config.yml");
+        final RepoConfig config = this.readMin();
         MatcherAssert.assertThat(
             "Unexpected custom config",
             config.settings().toCompletableFuture().get().isEmpty()
         );
     }
 
+    @Test
+    public void readContentLengthMax() throws Exception {
+        final RepoConfig config = this.readFull();
+        final long value = 123L;
+        MatcherAssert.assertThat(
+            config.contentLengthMax().toCompletableFuture().join(),
+            new IsEqual<>(Optional.of(value))
```

Comment:
@olegmoz let's compare the value directly by getting `Optional.value()` instead of creating `Optional.of()` in the assertion.

Reply:
@carlosmiranda it is better to compare `Optional`, not it's value here. If the `Optional` is absent, then we will receive assertion exception which is expected in test, not `IllegalStateException` on attempt to read it's value

Reply:
@olegmoz ok, that makes sense, let's keep it as is.